prql-compiler/src/sql/std.sql.prql

```diff
@@ -56,29 +57,33 @@ let neg = l -> null
 let not = l -> null
 
 module postgres {
-    let regex_search = text pattern -> s"{text} ~ {pattern}"
+    let div_i = l r -> s"trunc({l} / {r})::bigint"
+    let round = n_digits column -> s"ROUND(({column})::numeric, {n_digits})"
+    let regex_search = text pattern -> s"({text} ~ {pattern})"
 }
 
 module sqlite {
-    let regex_search = text pattern -> s"{text} REGEXP {pattern}"
+    let div_i = l r -> s"ROUND(ABS({l} / {r}) - 0.5) * SIGN({l} / {r})"
+    let regex_search = text pattern -> s"({text} REGEXP {pattern})"
 }
 
 module mssql {
     let regex_search = text pattern -> null
 }
 
 module mysql {
-    let regex_search = text pattern -> s"{text} REGEXP {pattern}"
+    let div_f = l r -> s"(CAST({l} AS FLOAT) / {r})"
+    let mod = l r -> s"FLOOR(MOD({l}, {r}))"
+    let regex_search = text pattern -> s"({text} REGEXP {pattern})"
 }
 
 module bigquery {
     let regex_search = text pattern -> s"REGEXP_CONTAINS({text}, {pattern})"
 }
 
 module duckdb {
-    let regex_search = text pattern -> s"REGEXP_MATCHES({text}, {pattern})"
-}
+    # TODO: uncomment when updated on latest DuckDB version
+    # let div_i = l r -> s"({l} // {r})"
```

Comment:
DuckDB's `//` does not work for floats, so I think we can't use this.
duckdb/duckdb#7558